### PR TITLE
[pulsar-io] [doc] Add docs for Redis sink

### PR DESCRIPTION
**Pull request by murong00**
### Motivation

Add docs for Redis sink.

### Modifications

Add `io-redis.md`.

### CODE REVIEW

File: site2/website/versioned_docs/version-2.3.1/io-redis.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+id: version-2.3.1-io-redis
```
Comment (sijie) [inline site2/website/versioned_docs/version-2.3.1/io-redis.md]: was redis connector included in 2.3.1? if it doesn't, we just need it documented in master.

Comment (murong00) [reply site2/website/versioned_docs/version-2.3.1/io-redis.md]: Not included yet, changed.

**Comment by merlimat** [comment]
run java8 tests
run cpp tests

**Comment by sijie** [comment]
run java8 tests

**Comment by sijie** [comment]
run java8 tests

**Comment by murong00** [comment]
run java8 tests

**Comment by murong00** [comment]
run java8 tests

**Pull request merged**